test/specs/repo/property.ts

```diff
@@ -58,21 +51,25 @@ describe( 'Property', () => {
 			} );
 
 			it( 'Should be able to add reference to property', async () => {
-				await Property.addReference.click();
+				await Property.addReferenceLink.click();
 				// fill out property id for reference
 				await $( '.ui-entityselector-input' ).isFocused();
 				await browser.keys( stringPropertyId.split( '' ) );
 				await $( propertyIdSelector( stringPropertyId ) ).click();
 				await browser.keys( 'REFERENCE'.split( '' ) );
-				await Property.saveStatement.click();
+				await Property.saveStatementLink.click();
 			} );
 
-			it( 'Should be able to see added reference', async () => {
+			it( 'Should be able to see added reference', async function () {
+				this.retries( 4 );
```

Comment:
Will retry this up to 4 times before calling it a failure. The number of retries, if more than 1, will be reported parenthetically by the Spec reporter at the end.  Note that this runs the `beforeTest` before retrying the test each time as well, so the page is reloaded each time.